core/src/main/java/module-info.java

```diff
@@ -564,7 +565,7 @@
             io.questdb.griffin.engine.functions.catalogue.ProcCatalogueFunctionFactory,
             io.questdb.griffin.engine.functions.catalogue.RangeCatalogueFunctionFactory,
             io.questdb.griffin.engine.functions.catalogue.PrefixedPgGetKeywordsFunctionFactory,
-            io.questdb.griffin.engine.functions.catalogue.TableMetadataCursorFactory,
+            TableListFunctionFactory,
```

Comment:
nit: should include package name to keep things consistent here
```suggestion
            io.questdb.griffin.engine.functions.catalogue.TableListFunctionFactory,
```